src/platform/backends/qemu/qemu_virtual_machine_factory.cpp

```diff
@@ -261,19 +261,12 @@ void mp::QemuVirtualMachineFactory::configure(const std::string& name, YAML::Nod
 
 void mp::QemuVirtualMachineFactory::check_hypervisor_support()
 {
-    QProcess cmd;
-    cmd.start("bash",
-              QStringList() << "-c"
-                            << "egrep -m1 -w '^flags[[:blank:]]*:' /proc/cpuinfo | egrep -wo '(vmx|svm)'");
-    cmd.waitForFinished();
-    auto virt_type = cmd.readAll().trimmed();
+    QProcess check_kvm;
+    check_kvm.start("check_kvm_support");
```

Comment:
How would this work when running from a local build?

Reply:
It won't work, but do we really care?  Local builds would be for developers, and developers should be able to figure if their machine currently supports kvm, right?

Reply:
Well, it will throw just below, no?

Reply:
No, the return value from bash of a missing binary is 127, so it will just basically be a no-op since I'm only checking a return value of 1.

Reply:
d'oh.